ontology/yaml/resources/subfields/subfields.yaml

```diff
@@ -136,6 +140,7 @@ descriptor:
   north: "Cardinal direction; opposite of south"
   occupancy: "State of being occupied."
   occupied: "State of being within scheduled (time of day or week) run-time parameters."
+  oil: "Oil in heater."
```

Comment:
Too specific to the application and references itself in its own definition; try this instead: "a viscous liquid derived from petroleum, especially for use as a fuel or lubricant"

Reply:
the description for oil has been changed